src/components/landing.js

```diff
@@ -27,9 +27,7 @@ export default function Landing (props) {
       </div>
     );
   } else {
-    let userIsAuthed = authMall.currentAuthService.isAuthed();
-
-    auth.setUserAuthedState(userIsAuthed, () => {
+    auth.setUserAuthedState(auth.userAuthed(), () => {
       if (authMall.isUserNew() == true) {
```

Comment:
Three errors showing up for the landing page (two unrelated to this PR, but figured we should review:

```
src/components/landing.js
  Line 1:17:   'useState' is defined but never used         no-unused-vars
  Line 6:8:    'ageConfirmStore' is defined but never used  no-unused-vars
  Line 31:32:  Expected '===' and instead saw '=='          eqeqeq
```

Reply:
yeah it's just defined but it's not being used. I could remove them for this PR.

Reply:
It looks like in this case we should be using the `===` right? Because authMall should be returning `true` or `false`. The other case seems less clear to me.